lib/script.js

```diff
@@ -1,21 +1,33 @@
 /**** To change time from UNIX to normal ***/
 // var time = new Date(timestamp*1000);
 
+var my_location = ""
+var my_city = ""
+var my_state = ""
 var weather_data = [];
```

Comment:
I see below that you are calling weather_data[0] pretty regularly. What if you just declared the variable without assigning it to an empty array, so that when you assign it further down the line, you do not always have to select the first element in the array? (continued on line 46)

Reply:
I think that makes more sense but I ran into some issues with scoping when trying to implement that :/